lib/BridgeList.php

```diff
@@ -74,7 +74,7 @@ private static function getBridges($showInactive, &$totalBridges, &$totalActiveB
 
 		foreach($bridgeList as $bridgeName) {
 
-			if($bridgeFac->isWhitelisted($bridgeName)) {
+			if($bridgeFac->isWhitelisted($bridgeName) && $bridgeFac->create($bridgeName)->isConfigurationValid()) {
```

Comment:
Bridges with invalid configurations are filtered from the whitelist, which could be confusing. Maybe it is okay to just keep show the bridge and have it fail once it gets used. What do you think?